app-cdr/ecm/ecm-1.0.0.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+inherit toolchain-funcs versionator
+
+MY_PV=$(delete_all_version_separators)
+
+DESCRIPTION="CD EDC/ECC code (un)stripper"
+HOMEPAGE="https://web.archive.org/web/20091104002036/www.neillcorlett.com/ecm"
+SRC_URI="https://web.archive.org/web/20091021035854/www.neillcorlett.com/downloads/${PN}${MY_PV}.zip"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~arm ~mips ~ppc ~ppc64 ~amd64-linux ~x86-linux ~x64-macos"
```

Comment:
Do you have Gentoo installs for all those systems? VMs count but qemu-user doesn't ;-).